Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -18,6 +18,8 @@
 #include "Core/HW/EXI/EXI.h"
 #include "Core/HW/Memmap.h"
 
+//#define BBA_TRACK_PAGE_PTRS
```

Comment:
This should be removed or added in Dolphin's config system.

Reply:
The #ifdef for this is preexisting so you don't have to convert this to a setting, but yeah probably remove this commented out line at least.